cmd/minishift/cmd/config/config.go

```diff
@@ -167,6 +167,9 @@ var (
 
 	// Systemtray
 	AutoStartTray = createConfigSetting("auto-start-tray", SetBool, []setFn{validations.IsSystemTrayAvailable}, nil, true, true)
+
+	// Static-IP
+	StaticIPAutoSet = createConfigSetting("static-ip", SetBool, nil, nil, true, nil)
```

Comment:
@praveenkumar This is not enabled by default anymore. Know  that you have handled the default switch scenario in case of HyperV, I think we can enable it by default. Also in this state it is not fixing the original issue i.e. `Apply --set-static option automatically after start of an instance/profile`.

Reply:
@gbraad Do you think now we should default enable it and this will not create any issue for any hypervisor.

Reply:
I think we can enable by default.